src/ert/config/queue_config.py

```diff
@@ -292,7 +295,10 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
         _grouped_queue_options = _group_queue_options_by_queue_system(
             _raw_queue_options
         )
-
+        activate_script = ErtPluginManager().activate_script()
```

Comment:
I think this behavior should be activated by the `with_plugins` functions on the ErtConfig and not directly in here :thinking:

Reply:
`with_plugins` is only for forward model steps, so would have to expand it. Think it is just intended as a convenience function to avoid having to start the plugin manager directly.

Reply:
If we do it like it is now there is no way of instantiating an ErtConfig that does not include plugins, if we have it in the `with_plugins` it's consistent when plugins are activated. This PR also adds to this functionality: https://github.com/equinor/ert/pull/9050/files

Reply:
Not sure I agree for all plugins, but moved it, as well as the default.